fuel/transformers/__init__.py

```diff
@@ -459,13 +484,13 @@ class MultiProcessing(Transformer):
 
     """
     def __init__(self, data_stream, max_store=100):
-        super(MultiProcessing, self).__init__(data_stream)
+        super(MultiProcessing, self).__init__(data_stream, batch_input=True)
         self.background = BackgroundProcess(data_stream, max_store)
         self.proc = Process(target=self.background.main)
         self.proc.daemon = True
         self.proc.start()
 
-    def get_data(self, request=None):
+    def get_data_from_batch(self, request=None):
```

Comment:
Since this one just passes on data, I think its agnostic to whether the input is a batch or single example. You can just leave the `get_data` method here.